templates/emails/plain/failed-refund-admin.php

```diff
@@ -0,0 +1,42 @@
+<?php
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+echo esc_html( $email_heading ) . "\n\n";
+
+printf(
+	// translators: 1) an order number, 2) the customer's full name, 3) the reason for the failure.
+	esc_html_x(
+		'The refund request for order %1$s from %2$s has failed. Reason: %3$s. The customer was also notified.',
```

Comment:
Another small issue: the reason values mostly include a trailing `.`, so we should not add another one.
```suggestion
		'The refund for order %1$s from %2$s has failed. Reason: %3$s The customer was also notified.',
```

Reply:
Since I decided to remove the `.` from the refund failure reason strings, I just updated the messaging here by removing the `request` word in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/4403/commits/719e420582fc65916cb9397f9649ec4db5083db1